client/cli/Cargo.toml

```diff
@@ -14,6 +14,7 @@ anyhow = "1.0"
 chrono = "0.4.7"
 tonic = "0.1"
 hex = "0.3.2"
+curl = "0.4.25"
```

Comment:
can we use https://docs.rs/reqwest/0.10.1/reqwest/ instead?  Mint code is already using it (due to its support on system proxies), it is best if we don't introduce new libraries.